.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm test
```

Comment:
I could be wrong but both `npm` and `npx` could be changed to `yarn run`.

I would make these both `yarn` because that's what most of our projects use.